spec/routing/project_routing_spec.rb

```diff
@@ -30,29 +30,29 @@
 
 describe ProjectsController, type: :routing do
   describe 'index' do
-    it { expect(get('/projects')).to route_to(controller: 'projects', action: 'index') }
-    it { expect(get('/projects.atom')).to route_to(controller: 'projects', action: 'index', format: 'atom') }
-    it { expect(get('/projects.xml')).to route_to(controller: 'projects', action: 'index', format: 'xml') }
+    it { expect(get('/projects')).to        route_to(controller: 'projects', action: 'index') }
+    it { expect(get('/projects.atom')).to   route_to(controller: 'projects', action: 'index', format: 'atom') }
+    it { expect(get('/projects.xml')).to    route_to(controller: 'projects', action: 'index', format: 'xml') }
   end
 
   describe 'show' do
-    it { expect(get('/projects/1')).to route_to(controller: 'projects', action: 'show', id: '1') }
-    it { expect(get('/projects/1.xml')).to route_to(controller: 'projects', action: 'show', id: '1', format: 'xml') }
-    it { expect(get('/projects/test')).to route_to(controller: 'projects', action: 'show', id: 'test') }
+    it { expect(get('/projects/1')).to      route_to(controller: 'projects', action: 'show', id: '1') }
+    it { expect(get('/projects/1.xml')).to  route_to(controller: 'projects', action: 'show', id: '1', format: 'xml') }
+    it { expect(get('/projects/test')).to   route_to(controller: 'projects', action: 'show', id: 'test') }
   end
 
   describe 'new' do
-    it { expect(get('/projects/new')).to route_to(controller: 'projects', action: 'new') }
+    it { expect(get('/projects/new')).to    route_to(controller: 'projects', action: 'new') }
   end
 
   describe 'create' do
-    it { expect(post('/projects')).to route_to(controller: 'projects', action: 'create') }
-    it { expect(post('/projects.xml')).to route_to(controller: 'projects', action: 'create', format: 'xml') }
+    it { expect(post('/projects')).to       route_to(controller: 'projects', action: 'create') }
```

Comment:
Unnecessary spacing detected.<br>Put one space between the method name and the first argument.